src/services/TestDataService.ts

```diff
@@ -2157,4 +2158,44 @@ export class TestDataService {
 
         return Object.assign({}, basicCmsPage, overrides);
     }
+
+    getBasicCustomerGroup(overrides: Partial<CustomerGroup>): Partial<CustomerGroup> {
```

Comment:
Please put the `get` method to the GET part of the TestDataService

Reply:
yes

Reply:
And you can rename the method with `getBasicCustomerGroupStruct(...)`

Reply:
I think it is in the correct place; I mean, the last part of the file belongs to the getxxxStruct() methods. I just renamed it as your suggestion.